.github/workflows/deploy-release-azure-webapps.yml

```diff
@@ -33,6 +33,11 @@ jobs:
       - name: Install Dependencies
         run: rush install
 
+      # Switch flavor to stable when it is release branch
+      - name: Switch flavor for stable build
+        if: ${{ !contains(github.ref, 'beta') }}
```

Comment:
This is brittle as it depends on the branch name directly. That might be surprising.

Can this workflow take an explicit argument instead to say which flavor build to deploy (with default to `beta`)?

Reply:
Agree, this is following how currently we do release for branches - but we do need to iterate that in following PRs